acceptance/tests/facts/partitions.rb

```diff
@@ -0,0 +1,37 @@
+test_name "C96148: verify partitions facts" do
+  tag 'risk:high'
+
+  confine :except, :platform => 'osx' # no partitions on osx
+  confine :except, :platform => 'windows' # no partitions on windows
+  confine :except, :platform => 'solaris' # no partitions on solaris
+
+  require 'json'
+
+  possible_facts = [
```

Comment:
Is there a reason that the expected facts in this test are implemented as arrays rather than as hashes like they are in the other facter tests. Should they be moved into `acceptance/lib/facter/acceptance/base_fact_utils.rb` with the other expected facts?